contracts/RLPReader.sol

```diff
@@ -1,58 +1,62 @@
 // SPDX-License-Identifier: Apache-2.0
 
 /*
-* @author Hamdi Allam hamdi.allam97@gmail.com
-* Please reach out with any questions or concerns
-*/
-pragma solidity >=0.5.0 <0.7.0;
+ * @author Hamdi Allam hamdi.allam97@gmail.com
+ * Please reach out with any questions or concerns
+ */
+pragma solidity ^0.8.15;
```

Comment:
This should be `>=0.5.0 <0.8.15`.

We'd like to not allow any version so that we can pre-emptively fix breaking changes